tests/cupyx_tests/scipy_tests/spatial_tests/test_distance.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+
+import numpy
+import cupy
+try:
+    import scipy.spatial  # NOQA
+    import scipy.spatial.distance  # NOQA
+    scipy_available = True
+except ModuleNotFoundError:
+    scipy_available = False
+import cupyx.scipy.spatial.distance  # NOQA
+try:
+    import pylibraft  # NOQA
+    pylibraft_available = True
+except ModuleNotFoundError:
+    pylibraft_available = False
+from cupy import testing
+
```

Comment:
I am guess Jenkins fails because the entire `cupyx.scipy.spatial.distance` support is missing on HIP. Perhaps we could add a module-level skip to check if `cupy.cuda.is_hip` is `True`?